setup.py

```diff
@@ -19,31 +22,41 @@ def get_version(filename):
 
 line = 'daffy'
 
+install_requires = [
+    'gym>=0.17.1',
+    'numpy>=1.10.0',
+    'pyglet<=1.5.0',
```

Comment:
Any reason for this ? I thought we had removed this

Reply:
removed what? 

(I just moved things around)